### PR TITLE
Added switch

### PR DESCRIPTION
**Pull request by cayhorstmann**

### CODE REVIEW

File: site/content/features/switch.md [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+---
+
+
+In preparation for a more general pattern matching construct, JEP 361 provides an expression that selects one of multiple values, given an operand that is an integer, string, or `enum` constant, using the familiar `switch` syntax The JEP also provides additional forms of `switch` expressions and statements, in order to make the feature more regular. The final release occurred in JDK 14.
+
+## The Classic Switch Statement
+
+The C language introduced a `switch` statement which is quite different from the more orderly control flow constructs (`if`, `while`, and so on). Execution jumps to a labeled branch, and falls through subsequent branches, unless explicitly terminated with a `break`. 
+
+Here is the classic example from Kernighan & Ritchie:
```
Comment (marchof) [inline site/content/features/switch.md:16]: What about a link to the book on Wikipedia?

```suggestion
Here is the classic example from [Kernighan & Ritchie](https://en.wikipedia.org/wiki/The_C_Programming_Language):
```

File: site/content/features/switch.md [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+    case ' ':
+    case '\n':
+    case '\t':
+        nwhite++;
+        break;
+    default:
+        nother++;
+        break;
+}
+```
```
Comment (marchof) [inline site/content/features/switch.md:33]: I think those ticks should be removed now that you use html.
```suggestion
```

File: site/content/features/switch.md [hunk 3]
```diff
@@ -0,0 +1,319 @@
+---
+title: Switch Expressions (JEP 361)
+copyright: Cay S. Horstmann 2021. All rights reserved.
+jep: 361
+jdkversion: 14
+type: "sandbox"
+---
+
+
+In preparation for a more general pattern matching construct, JEP 361 provides an expression that selects one of multiple values, given an operand that is an integer, string, or `enum` constant, using the familiar `switch` syntax The JEP also provides additional forms of `switch` expressions and statements, in order to make the feature more regular. The final release occurred in JDK 14.
```
Comment (marchof) [inline site/content/features/switch.md:10]: Period missing after "syntax"?
```suggestion
In preparation for a more general pattern matching construct, JEP 361 provides an expression that selects one of multiple values, given an operand that is an integer, string, or `enum` constant, using the familiar `switch` syntax. The JEP also provides additional forms of `switch` expressions and statements, in order to make the feature more regular. The final release occurred in JDK 14.
```

Comment (marchof) [inline site/content/features/switch.md:10]: Link JEP
```suggestion
In preparation for a more general pattern matching construct, [JEP 361](https://openjdk.java.net/jeps/361) provides an expression that selects one of multiple values, given an operand that is an integer, string, or `enum` constant, using the familiar `switch` syntax The JEP also provides additional forms of `switch` expressions and statements, in order to make the feature more regular. The final release occurred in JDK 14.
```

File: site/content/features/switch.md [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+        };
+        System.out.println(seasonName);
+    }
+}
+{{< /sandboxsource >}}
+{{< /sandbox >}}
+
+Just like `return` exits a method with a value, `yield` exits the block with a value. 
+
+Prior versions (JEP 325 and 354) used the `break` keyword for the same purpose, but this was deemed to be confusing when previewed in JDK 12 and 13.
```
Comment (marchof) [inline site/content/features/switch.md:121]: Link JEPs?
```suggestion
Prior versions ([JEP 325](https://openjdk.java.net/jeps/325) and [354](https://openjdk.java.net/jeps/354)) used the `break` keyword for the same purpose, but this was deemed to be confusing when previewed in JDK 12 and 13.
```

File: site/content/features/switch.md [hunk 5]
```diff
@@ -0,0 +1,319 @@
...
+
+1. There are four forms of `switch`: statements and expressions, with and without fall through
+2. A `switch` expression selects one or more values, matching an operand. 
+3. The cases in a `switch` expression must be disjoint and exhaustive
+4. A `switch` expression or statemnt with `->` tokens does not have fall through behavior
+5. A `case` label can be followed by multiple constants, separated by commas
+6. In a `switch` expression, the `yield` statement exits and yields a value
+7. In a `switch` expression without fall through, the `->` token can be followed by a block statement that yields a value
+
+
```
Comment (marchof) [inline site/content/features/switch.md]: Should we add a references section as with the other articles?
```suggestion
## References

* [JEP 325: Switch Expressions (Preview), OpenJDK](https://openjdk.java.net/jeps/325)
* [JEP 354: Switch Expressions (Second Preview), OpenJDK](https://openjdk.java.net/jeps/354)
* [JEP 361: Switch Expressions, OpenJDK](https://openjdk.java.net/jeps/361)
```

File: site/content/features/switch.md (hunk 3 above)
Comment (cayhorstmann) [reply site/content/features/switch.md]: I like the reference section better.

File: site/content/features/switch.md (hunk 5 above)
Comment (cayhorstmann) [reply site/content/features/switch.md]: Yes, we should. I can't commit this from here--it's "outdated". I pushed again with the reference section.

**Pull request merged**